test/app-files-updates.ts

```diff
@@ -71,7 +71,7 @@ describe("App files copy", () => {
 
 	it("copies all app files but app_resources when not bundling", () => {
 		const updater = new CopyAppFilesUpdater([
-			"file1", "dir1/file2", "App_Resources/Android/blah.png"
```

Comment:
why is this changed? If the behavior is changed, the testname and expected results should be changed

Reply:
@rosen-vladimirov that's an oversight on my part. I changed the only failing test, because the app_resources were no longer filtered as part of the `copy()` method but in [public updateApp()](https://github.com/NativeScript/nativescript-cli/pull/3403/files#diff-d7315453ac31262b45d0c2530d69cd94R14) which returns void and I found no straightforward way of mocking it in order to update the test properly.